modules/external-auth-fields.adoc

```diff
@@ -28,6 +37,22 @@ The following table lists all available OIDC provider parameters for direct auth
 |`claimMappings.groups.prefix`
 |Configures the prefix that is applied to the cluster identity attribute during the process of mapping JWT claims to cluster identity attributes.
 
+|`claimMappings.uid`
+|An optional field for configuring the claim mapping used to construct the uid for the cluster identity.
```

Comment:
This is also missing some useful context for end users:

>	// When using uid.claim to specify the claim it must be a single string value.
	// When using uid.expression the expression must result in a single string value.
	//
	// When omitted, this means the user has no opinion and the platform
	// is left to choose a default, which is subject to change over time.
	// The current default is to use the 'sub' claim.